packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -159,7 +159,44 @@ export class PipelineService {
     ).then(response => {
       let dialogTitle;
       let dialogBody;
-      if (response['run_url']) {
+      if (response['git_url']) {
```

Comment:
This seems to introduce a lot of duplicated code.  SInce the presence of `git_url` is a subset of the presence of `run_url`, it seems like the original block (`if response['run_url']`) could still exist, with a sub-condition that checks for `git_url` to produce the git-related destination.

Also, is there something else in `response` (or elsewhere) that indicates the platform (e.g., 'airflow', 'kfp')?  If not, I think we should probably add some indicator so that we can remove the assumption that `git_url` is Airflow.

Reply:
I agree, we should nest the relevant new code inside the existing if block, I'll detail below

Reply:
Updated with `response`json  with `platform` field